serving/docker/partition/utils.py

```diff
@@ -72,7 +62,7 @@ def get_djl_version_from_lib():
 
 
 def is_engine_mpi_mode(engine):
```

Comment:
do we use this function anywhere currently?

Reply:
No, not right now. But if we unify TRTLLM which is in a separate script to this partition script, then we need this.